frontend/public/components/api-explorer.tsx

```diff
@@ -111,7 +111,7 @@ const Group: React.FC<{ value: string }> = ({ value }) => {
   ) : (
     <>
       {first}
-      <span className="text-muted">.{rest.join('.')}</span>
+      <p><span className="text-muted">.{rest.join('.')}</span></p>
```

Comment:
We should avoid a paragraph break in the middle of the group. Can we use `co-break-word` or `co-break-all` instead to make sure the group wraps?

cc @sg00dwin @rhamilto

Reply:
Agreed.  I'd try:
```
<span className="co-break-word">
  {first}
  <span className="text-muted">.{rest.join('.')}</span>
</span>

Reply:
> We should avoid a paragraph break in the middle of the group. Can we use `co-break-word` or `co-break-all` instead to make sure the group wraps?
> 
> cc @sg00dwin @rhamilto

I got it.I'm not resolved it actually.

Reply:
> Agreed. I'd try:
> 
> ```
> <span className="co-break-word">
>   {first}
>   <span className="text-muted">.{rest.join('.')}</span>
> </span>
> ```

Can it works?